File: src/VRE.Vridge.API.Client/Messages/v1/Control/Responses/APIStatus.cs
using System.Collections.Generic;

namespace VRE.Vridge.API.Client.Messages.v1.Control.Responses
{
    /// <summary>
    /// Contains list of available (or not) API services.
    /// </summary>
    public class APIStatus : ControlResponseHeader
    {        
        public List<EndpointStatus> Endpoints;        
    }
}

File: src/java/VRE.Vridge.API/VRE.Vridge.API.Client/src/main/java/com/riftcat/vridge/api/client/java/control/responses/APIStatus.java
package com.riftcat.vridge.api.client.java.control.responses;

import com.riftcat.vridge.api.client.java.EndpointNames;

import java.util.List;

public class APIStatus {

    public List<EndpointStatus> Endpoints;

    @Override
    public String toString() {

        String strStatus = "";

        for(EndpointStatus status : Endpoints){
            strStatus += status.Name + "(" + status.codeString() + ") | ";
        }

        return strStatus;
    }

    public EndpointStatus findEndpoint(String name){
        for (EndpointStatus endpoint : Endpoints) {
            if(endpoint.Name.equals(name)){
                return endpoint;
            }
        }

        return null; }

}
